bridge/engine/EngineMixer.cpp

```diff
@@ -1839,14 +1980,38 @@ void EngineMixer::onRtcpPacketDecoded(transport::RtcTransport* sender,
 SsrcOutboundContext* EngineMixer::obtainOutboundSsrcContext(size_t endpointIdHash,
     concurrency::MpmcHashmap32<uint32_t, SsrcOutboundContext>& ssrcOutboundContexts,
     const uint32_t ssrc,
-    const RtpMap& rtpMap)
+    const RtpMap& rtpMap,
+    const bool ssrcRewrite)
 {
     auto ssrcOutboundContextItr = ssrcOutboundContexts.find(ssrc);
     if (ssrcOutboundContextItr != ssrcOutboundContexts.cend())
     {
+        if (ssrcOutboundContextItr->second.markedForDeletion)
+        {
+            // The outboundContext is marked for deletion after owner inbound stream
```

Comment:
I can see in code it is marked for deletion. Hopefully it is clear from other code that the remove job has been posted, but if we want to mention it reduntantly here the comment could be.
"RemoveOutboundSsrc job posted.
Do not use this outbound context."

But even that is evident from what the code does, return nunllptr and exit here.